retrieverdash/requirements.txt

```diff
@@ -1,5 +1,32 @@
-Django==2.2.13
+argcomplete==1.12.1
+asgiref==3.2.10
+attrs==20.2.0
+certifi==2020.6.20
+chardet==3.0.4
+Django==3.1.2
+django-crontab==0.7.1
+future==0.18.2
+h5py==2.10.0
+idna==2.10
+importlib-metadata==2.0.0
+iniconfig==1.1.1
+numpy==1.19.2
+packaging==20.4
+pandas==1.1.3
+Pillow==8.0.0
+pluggy==0.13.1
+py==1.9.0
+pyparsing==2.4.7
+pytest==6.1.1
+pytest-django==4.0.0
+python-dateutil==2.8.1
 pytz==2017.3
-git+https://www.github.com/weecology/retriever@master
-pytest-django
-django-crontab
+requests==2.24.0
+retriever
```

Comment:
I think we may still need the `dev` version here vs the `release `"git+https://www.github.com/weecology/retriever@master"

What do you think?